pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -246,11 +246,11 @@ let
     "--ghc-options=-haddock"
   ];
 
-  postPhases = optional doInstallIntermediates [ "installIntermediatesPhase" ];
+  postPhases = optional doInstallIntermediates "installIntermediatesPhase";
```

Comment:
Is this semantically a no-op?

Reply:
it basically changes the value from `[ [ "installIntermediatesPhase" ] ]` to `[ "installIntermediatesPhase" ]`, so it should be a no-op since it gets turned into a string